ui/packages/app/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "mlp-ui",
-  "version": "1.7.0",
+  "version": "1.7.6",
   "private": true,
   "license": "Apache-2.0",
   "dependencies": {
+    "@caraml-dev/ui-lib": "^1.7.5-build.5-59f13e1",
```

Comment:
I guess we can't use `@caraml-dev/ui-lib` @ `v1.7.6` until it's published. If so, should we update the MLP app UI in a separate PR after the lib is released?

Reply:
If we don't point this to the dev build, the CI will fail because there's no such version `v1.7.0`.

Reply:
Ok.. I assume that after we release `v1.7.6`, we will update the MLP UI app deps again? If so, then LGTM.

Reply:
Yes that's correct!